pyigm/surveys/igmsurvey.py

```diff
@@ -474,7 +474,9 @@ def ions(self, Zion, Ej=0., skip_null=True):
         Ej : float [1/cm]
            Energy of the lower level (0. is resonance)
         skip_null : boolean (False)
-           Skip systems without an entry, else pad with zeros 
+           Skip systems without an entry, else pad with zeros
+        pad_with_nulls : bool, optional
+           Pad missing/null systems with empty values.  A bit risky
```

Comment:
Why is this risky?   Perhaps it's due to my ignorance, but dealing with tables having null values is much less fussy than masked tables.

Reply:
The main thing I was reacting against was hardcoding what the Table 
format will look like.  I can imagine it changing and then this will break.